jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/FrameFlusher.java

```diff
@@ -113,6 +114,7 @@ private Action flush()
         {
             if (!BufferUtil.isEmpty(aggregate))
             {
+                aggregate.flip();
```

Comment:
can you describe those side effects?   lets fix buffer util rather than mixing styles.